### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by ovcOS**

### CODE REVIEW

File: app/assets/stylesheets/pages/_marketplace.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  color: white;
+  background-image: linear-gradient(rgba(0,0,0,0.2) 0%, rgba(0,0,0,0.8) 70%);
+}
+
+.card__profile-photo {
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+  filter: opacity(70%);
+
```
Comment (coryb2424) [inline app/assets/stylesheets/pages/_marketplace.scss]: space

File: app/assets/stylesheets/pages/_marketplace.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+.card__profile-info {
+  height: 30%;
+  position: absolute;
+  left: 10px;
+  bottom: 15px;
+
```
Comment (coryb2424) [inline app/assets/stylesheets/pages/_marketplace.scss]: space

File: app/assets/stylesheets/pages/_marketplace.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+.card__skills {
+  padding: 10px 0px 0px 10px;
+  box-shadow: 0.2px 0.2px 4px lightgray;
+  color: $gunmetal;
+
```
Comment (coryb2424) [inline app/assets/stylesheets/pages/_marketplace.scss]: space

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -2,7 +2,7 @@ class UsersController < ApplicationController
   before_action :find_user, only: %i[show edit]
 
   def index
-    @users = User.all
+    @users = User.all.limit(5)
```
Comment (coryb2424) [inline app/controllers/users_controller.rb]: @users = User.best_matches(current_user)

File: app/assets/stylesheets/pages/_marketplace.scss (hunk 1 above)
Comment (ovcOS) [reply app/assets/stylesheets/pages/_marketplace.scss]: done

File: app/assets/stylesheets/pages/_marketplace.scss (hunk 2 above)
Comment (ovcOS) [reply app/assets/stylesheets/pages/_marketplace.scss]: done

**Pull request merged**